app/views/issues/show.html.erb

```diff
@@ -33,10 +33,11 @@
             <div class="note-text-inner">
               <h4 class="mb-4 header-underline">
                 <span class="text-truncate" title="<%= @issue.title %>"><%= @issue.title %></span>
+                <%= render partial: 'shared/liquid_loading' %>
                 <%= render partial: 'actions' %>
               </h4>
-              <div class="content-textile" data-behavior="content-textile">
-                <%= markup(@issue.text, liquid: true) %>
+              <div class="content-textile" data-behavior="content-textile liquid-async" data-path="<%= preview_project_issue_path(current_project, @issue) %>" data-content="<%= @issue.text %>" >
```

Comment:
ditto re: escaping.

Perhaps even extracte the :content_tag call to a helper that does all the data- stuff by receiving a path and context. Perhaps not needed.